esx_service/utils/auth_data_test.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+# Tests for auth.py
+
+import unittest
+import os
+import os.path
+import auth_data
+import uuid
+
+class TestAuthDataModel(unittest.TestCase):
+    """
+    Test the Authorization data model via the AuthorizationDataManager
+    """
+    db_path = "/tmp/test-auth.db"
```

Comment:
this is another place where you need to call the wrapper for DB name, something like get_auth_db_name().